### PR TITLE
REST > Settings getMany/getByIdentifier/getOne/create endpoints + doc

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

**Comment by ClaudeROSSI** [comment]
> @ClaudeROSSI Could you please check this one?

je regardes ca now

File: test/api/client/SettingApi.ts [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+    return super.readById(id, this.buildRestEndpointUrl(ServerRoute.REST_SETTING, { id }));
+  }
+
+  public async readByIdentifier(params?) {
+    return super.read(params, this.buildRestEndpointUrl(ServerRoute.REST_SETTINGS));
   }
 
   public async readAll(params?, paging = TestConstants.DEFAULT_PAGING, ordering = TestConstants.DEFAULT_ORDERING) {
-    return super.readAll(params, paging, ordering, '/client/api/Settings');
+    return super.readAll(params, paging, ordering, this.buildRestEndpointUrl(ServerRoute.REST_SETTINGS));
```
Comment (ClaudeROSSI) [inline test/api/client/SettingApi.ts:20]: Hi,

I have some billing tests that are not running properly anymore.
Please have a look to the screenshot below.
The billing tests are forcing the billing settings using the REST endpoints ... it seems that readdall does not return the expected data.

Screenshot from 2021-11-17 18-09-45

Please do not forget:
- to create the test context again (npm run mochatest:createContext)
- to refresh the test list (the billing tests have been reorganized recently).

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
billing tests are failing:

please have a look to my comments

**Comment by LucasBrazi06** [comment]
@ClaudeROSSI To be rechecked I think.
Thanks!!!

**Review by ClaudeROSSI — COMMENTED** [review]
@Ninjeneer : Smart Charging Tests are not running.

They are all green when using the master branch. 
Can you double check?

**Comment by ClaudeROSSI** [comment]
@Ninjeneer : Smart charging tests are failing after merging the master-qa branch

Screenshot from 2021-11-19 12-03-42

please couble check.
Thanks
Claude

**Review by ClaudeROSSI — APPROVED** [review]
All tests are now running properly

**Comment by LucasBrazi06** [comment]
> All tests are now running properly

Thanks Claude!!!

**Pull request merged**